lib/ProductOpener/Users.pm

```diff
@@ -1051,6 +1072,38 @@ sub init_user ($request_ref) {
 		);
 	}
 
+	# User was authenticated via OIDC
+	elsif ((defined $request_ref->{oidc_user_id}) and ($request_ref->{oidc_user_id} ne '')) {
+		$user_id = $request_ref->{oidc_user_id};
+
+		$log->context->{user_id} = $user_id;
+		$log->debug("user_id is defined") if $log->is_debug();
+		my $session = undef;
+
+		# If the user exists
+		if (defined $user_id) {
+
+			my $user_file = "$BASE_DIRS{USERS}/" . get_string_id_for_lang("no_language", $user_id) . ".sto";
+
+			if (-e $user_file) {
+				$user_ref = retrieve($user_file);
+				$user_id = $user_ref->{'userid'};
+				$log->context->{user_id} = $user_id;
+
+				if (not defined request_param($request_ref, 'no_log')) # no need to store sessions for internal requests
```

Comment:
I stole it from here 😅 https://github.com/openfoodfacts/openfoodfacts-server/blob/acf96bb5625f02ea1bc3bc0bcbdb82153a896517/lib/ProductOpener/Users.pm#L1111

Unless I'm mistaken, the idea is to not store the session in the `sto` file if they present the user+password, or an OIDC token (for the API request), because they won't use the session cookie in that way, anyways.